lib/modules/userTagger.js

```diff
@@ -158,9 +165,9 @@ export class Tag {
 				<a
 					class="userTagLink ${(text || color) ? 'hasTag' : 'RESUserTagImage'}"
 					${color && string._html`style="background-color: ${color}; color: ${bgToTextColorMap[color]} !important;"`}
-					title="set a tag"
+					title="${(text) ? text : 'set a tag'}"
```

Comment:
This could be simply `text || 'set a tag'`

Reply:
Thanks! I didn't notice that, I can make that change!